app/models/miq_group.rb

```diff
@@ -49,36 +49,30 @@ def self.next_sequence
   end
 
   def self.seed
-    role_map_file = File.expand_path(File.join(FIXTURE_DIR, "role_map.yaml"))
+    role_map_file = FIXTURE_DIR.join("role_map.yaml")
+    role_map = YAML.load_file(role_map_file) if role_map_file.exist?
+    return unless role_map
```

Comment:
I'll look, but the last time I checked a path to the fixture is being used in different ways across the app :confused: It's just candy at that point and I'd rather just move seeds entirely away from the model anyway...